ptbcontrib/username_to_chat_api/requirements.txt

```diff
@@ -1 +1,2 @@
-python-telegram-bot>=13.0,<20
\ No newline at end of file
+python-telegram-bot==20.0a0
+httpx ~= 0.22.0
```

Comment:
if that is ever changed by ptb, this would break though

Reply:
OTOH, we're already updating httpx to ~= 0.23.0, so having `httpx ~= 0.22.0` here would prevent install. I say better to keep it managed upstream, or at least remove/loosen the version requirement

Reply:
since `username_to_chat_api` explicitly uses httpx, IMO it's the right thing to explicitly list it here as well. Any update to the requirements ofc has to make sure that it doesn't cause conflicts

Reply:
20.0a0 requires httpx ~=  0.22.0. https://github.com/python-telegram-bot/python-telegram-bot/pull/3069 will released only in 20.0a1 …